Libraries/Text/Text/RCTTextView.m

```diff
@@ -521,8 +521,8 @@ - (void)copy:(id)sender
   NSData *rtf = [attributedText dataFromRange:NSMakeRange(0, attributedText.length)
                            documentAttributes:@{NSDocumentTypeDocumentAttribute: NSRTFDTextDocumentType}
                                         error:nil];
-#if TARGET_OS_IPHONE // [TODO(macOS GH#774)
-  NSMutableDictionary *item = [NSMutableDictionary new]; // TODO(macOS GH#774)
+#if TARGET_OS_IPHONE // [macOS
+  NSMutableDictionary *item = [NSMutableDictionary new]; // [macOS]
```

Comment:
I changed the check from `TARGET_OS_IPHONE` to `!TARGET_OS_OSX` as is in the rest of this PR / repo, and updated tag. The inner tag is a little confusing, but I think there's one because the initialization of `item` got moved from the original code to support the if/else.